docs/fsharp/tutorials/getting-started/getting-started-command-line.md

```diff
@@ -27,7 +27,7 @@ This article assumes that you know how to use a command line and have a preferre
 1. Open a Command Line/Terminal.
 2. Create a new directory named `FSNetCore`.  Open Visual Studio code or your preferred editor inside this directory. 
 3. Under `FSNetCore`, create `src` and `test` directories.
-4. Under `FSNetCore`, create a new file called `global.json`.  It should have this as its contents:
+4. Under `FSNetCore`, create a new file called `global.json`.  It should have the following contents:
```

Comment:
this seems wrong now. global.json now only supports specifying the sdk version. you should probably replace these steps with dotnet new sln and dotnet sln add